### PR TITLE
Update etcd chatflow with more info

### PR DESCRIPTION
**Pull request by MElborolossy**
### Description

Update etcd chatflow with more info related to certificate and add additional msg shown only if cert not verified, This message contains another method to use etcd using kubectl.

### Initialize step
image

### In case cert verified
withoutwarning
### In case cert not verified
image

### Checklist

- [x] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [x] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

File: jumpscale/packages/vdc_dashboard/chats/etcd.py [hunk 1]
```diff
@@ -69,17 +70,56 @@ def initializing(self):
             self.k8s_client.delete_deployed_release(self.release_name)
             self.stop(dedent(stop_message))
 
+        self.is_certified = False
+        start_request_time = time()
```
Comment (waleedhammam) [inline jumpscale/packages/vdc_dashboard/chats/etcd.py:74]: maybe we can replace this with to be more readable
```suggestion
        start_request_time = j.data.time.now().timestamp
```

File: jumpscale/packages/vdc_dashboard/chats/etcd.py [hunk 2]
```diff
@@ -69,17 +70,56 @@ def initializing(self):
...
+        request = None
+        j.logger.debug("ETCD Check certificate")
+        while time() - start_request_time <= timeout:
```
Comment (waleedhammam) [inline jumpscale/packages/vdc_dashboard/chats/etcd.py:77]:
```suggestion
        while j.data.time.now().timestamp - start_request_time <= timeout:
```

File: jumpscale/packages/vdc_dashboard/chats/etcd.py [hunk 3]
```diff
@@ -69,17 +70,56 @@ def initializing(self):
...
+                    ),
+                    md=True,
+                )
+                gevent.sleep(10)
+
+        if not self.is_certified:
+            request = j.tools.http.get(f"https://{self.domain}", timeout=timeout, verify=False)
+
+        # Etcd using TCP, always response return this msg "404 page not found"
+        if not "404 page not found" in str(request.content):
```
Comment (waleedhammam) [inline jumpscale/packages/vdc_dashboard/chats/etcd.py:95]: we can check for the status_code instead, to be 404 ?

Comment (MElborolossy) [reply jumpscale/packages/vdc_dashboard/chats/etcd.py:95]: It is preferred to let it check with content, as this response will be different if it fails for anything else.

**Review by waleedhammam — APPROVED** [review]

**Pull request merged**